sdk/tables/azure-data-tables/tests/_shared/testcase.py

```diff
@@ -73,12 +74,12 @@ def account_url(self, account, endpoint_type):
             if endpoint_type == "table":
                 return account.primary_endpoints.table.rstrip("/")
             if endpoint_type == "cosmos":
-                return "https://{}.table.cosmos.azure.com".format(account.name)
+                return "https://{}.table.{}".format(account.name, os.getenv("TABLES_COSMOS_ENDPOINT_SUFFIX"))
```

Comment:
When I ran these tests on my end, I initially got failures because I didn't have this environment variable set (so, request URLs had a suffix of `.table.none`). We'll need to add a default value that matches the public cloud suffix, like:
```python
cosmos_suffix = "cosmos.azure.com"
return "https://{}.table.{}".format(
    account.name, os.getenv("TABLES_COSMOS_ENDPOINT_SUFFIX", cosmos_suffix)
)
```

Reply:
I've added the env variable to test-resources.json: https://github.com/Azure/azure-sdk-for-python/blob/60e871aea918abaedceb71cfec7a951431727b19/sdk/tables/test-resources.json#L136-L139 So it should work with the new script.

Reply:
The environment variable outputs in `test-resources.json` are only registered in the test environment when the script is run, i.e. when live test resources are deployed. We don't deploy test resources for playback tests (like in CI pipelines), so these variables won't have a value in those pipeline tests